docs/source/joblib.rst

```diff
@@ -52,6 +52,15 @@ validated parameter search as follows.
    with parallel_backend('dask.distributed', scheduler_host='localhost:8786'):
        search.fit(digits.data, digits.target)
 
+## Nested Parallelism
+
+Note that this code will not work if you distribute it via dask in order to implement nested parallelism. The reason is that each worker will have a fresh copy of sklearn and `from sklearn.externals.joblib import parallel_backend` will not apply to them, resulting in a "KeyError" when trying to distribute, but no problem when running the inner loop locally. The solution is 
+
+.. code-block:: python
+
+    from distributed.joblib import DistributedBackend 
+    register_parallel_backend('distributed', DistributedBackend, make_default=True)
```

Comment:
Sure, the code that I used for dask(outer)/dask(inner) was 

dask_client = Client()
dask_addr = dask_client.scheduler.address

```
def function(i):
    parallel.register_parallel_backend('dask.distributed',DistributedBackend, make_default=True )
    with parallel_backend('dask.distributed', scheduler_host=dask_addr):
        score = sk.model_selection.cross_val_score(sk.svm.SVR(kernel='linear') , x , y , cv=2, n_jobs=-1)
    return score 

dask_client.gather(  dask_client.map(function, range(5))   )
```